Program.cs

```diff
@@ -1,4 +1,97 @@
-﻿using System;
-Console.WriteLine ("Hello, World!");
+﻿// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023.
+// Copyright (c) Metamation India.
+// ----------------------------------------------------------------------------
+// Program.cs
+// A10.2
+// Program to implement a double ended queue.
+// This queue can enqueue as well as dequeue elements from both the ends.
+// ------------------------------------------------------------------------------------------------

```

Comment:
Implement:

- PeekFront => Returns the first element added to the front of the deque
- PeekRear => Returns the first element added to the rear of the deque